lib/timeline/component/ItemSet.js

```diff
@@ -2070,69 +2084,65 @@ ItemSet.prototype._onAddItem = function (event) {
 
   var me = this;
   var snap = this.options.snap || null;
-  var item = this.itemFromTarget(event);
```

Comment:
Heh. I see. First of all `item` not used in this code block. Second, passing an existing item is silly since you're creation a new one.